department-of-veterans-affairs/abd-vro
# rm unused redis

## What was the problem?

Redis was being configured, maintained, and deployed; however, it was not being used by any of our services.

To the reviewer, note there are no removals of any Redis client code within our applications as evidence that it was not being used.

Associated tickets or Slack threads:
- #2697 

## How does this fix it?[^1]

Unlike other PRs for this ticket, this does not correspond to the creation of a new repo. This is because LHDI has enabled the use of ElastiCache so that in the future, if we would like to use a Redis-like DataStore, it is not longer necessary for us to maintain our own instance. 

## How to test this PR
- Remove deployments from escalating environments
- Local successful build and testing
- deployments of applications which had previously imported the Redis environment variables


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
